docs/developer-docs/identity/verifiable-credentials/how-it-works.mdx

```diff
@@ -77,6 +77,14 @@ From the developer’s perspective, the interaction between dapps uses window me
 
 For more details, check out the [specification in Internet Identity’s repository](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).
 
+## Test verifiable credentials
+
+There is a demo relying party deployed on ICP available for testing: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).
+
+You first need to login with an identity provider. To avoid using real users, it is recommended to use the staging instance of Internet Identity: [https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/](https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/).
+
+There is a demo issuer deployed on ICP  available for testing. It will issue any requested credential. Feel free to use it when testing the flow: [qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
```

Comment:
```suggestion
There is a demo issuer deployed on ICP available for testing. It will issue any requested credential. Feel free to use it when testing the flow: [qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
```